### PR TITLE
Migration logs in `cypress` package

### PR DESCRIPTION
**Pull request by callingmedic911**
Closes PRO-702.

### CODE REVIEW

File: packages/cypress/src/junit.ts [hunk 1]
```diff
@@ -130,7 +131,7 @@ function appendReplayUrlsToFailureNodes(node: INode, replayUrls: string[]) {
       failure.children[0] = escapeForXml(output);
     });
   } catch (e) {
```
Comment (bvaughn) [inline packages/cypress/src/junit.ts:133]: These `e` and `err` names are pet peeves of mine 😆 especially when they cause us to spend more characters renaming them

**Review by bvaughn — APPROVED** [review]

File: packages/cypress/src/fixture.ts [hunk 2]
```diff
@@ -16,9 +14,11 @@ function getFixtureFile() {
 }
 
 export function initFixtureFile() {
-  debug("REPLAY_CYPRESS_UPDATE_FIXTURE: %s", process.env.REPLAY_CYPRESS_UPDATE_FIXTURE);
+  logger.info("CypressReporter:InitFixtureFile", {
```
Comment (miriambudayr) [inline packages/cypress/src/fixture.ts:17]: Won't it already be clear from passing the package name/version to the `initLogger` call that the logs are coming from the cypress package?

Usually when I format logs, I do something more like:

`InitFixtureFile:Started`

That way, it is clear which function each log comes from, and in most cases there is a verb there so it is easy to read and understand without looking at the code.

Comment (miriambudayr) [reply packages/cypress/src/fixture.ts:17]: Examples: `FunctionName:Failed` or`FunctionName:Started` or `FunctionName:WillGetSources`, etc.

File: packages/cypress/src/steps.ts [hunk 3]
```diff
@@ -234,7 +230,7 @@ function groupStepsByTest(tests: Test[], steps: StepEvent[]): Test[] {
 
           // TODO [ryanjduffy]: Skipping handling after each events for now
           if (step.test[0] === AFTER_EACH_HOOK) {
-            debug("After each hooks are not currently supported");
+            logger.info("CypressReporter:AfterEachNotSupported");
```
Comment (miriambudayr) [inline packages/cypress/src/steps.ts:233]: Here I'd probably do `ShouldSkipstep:AfterEachNotSupported`

**Pull request merged**